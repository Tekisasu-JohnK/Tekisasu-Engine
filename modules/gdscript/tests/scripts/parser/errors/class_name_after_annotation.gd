# Error here. Annotations should be used before `class_name`, not after.
class_name HelloWorld
@icon("res://path/to/optional/icon.svg")

func test():
	pass
