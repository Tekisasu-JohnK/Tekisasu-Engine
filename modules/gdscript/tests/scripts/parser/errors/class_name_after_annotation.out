GDTEST_PARSER_ERROR
Annotation "@icon" is not allowed in this level.
